tests/cover_crypt.test.ts

```diff
@@ -360,11 +360,144 @@ test("Demo using KMS", async () => {
   // newConfidentialMkgCiphertext
   try {
     // will throw
-    new CoverCryptHybridDecryption(oldConfidentialMkgUserKey.bytes()).decrypt(
+    let x = new CoverCryptHybridDecryption(
+      oldConfidentialMkgUserKey.bytes(),
+    ).decrypt(newConfidentialMkgCiphertext)
+    console.log(new TextDecoder("utf-8").decode(x.plaintext))
+  } catch (error) {
+    // ==> the non rekeyed key cannot decrypt the new message
```

Comment:
expect that an error has been raised so? 
Maybe log error - a bit weird to do noghing in catch